apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/AnyAssertionSamples.kt

```diff
@@ -3,9 +3,20 @@
 
 package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
 
-import ch.tutteli.atrium.api.infix.en_GB.toBe
 import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.isNotSameAs
```

Comment:
those imports shouldn't be necessary if we have already the  star import, no? (`import ch.tutteli.atrium.api.infix.en_GB.toBe` was previously unnecessary as well)